client/src/components/reviews/AddReview.jsx

```diff
@@ -66,14 +44,17 @@ const AddReview = ({ productId, showModal, openModal, fetchData }) => {
       .then((res) => {
         alert('Your review has been submitted');
         openModal();
-        fetchData(productId);
+        fetchReviews(productId, reviewsSort)
+          .then((res) => { setReviews(res.data); });
```

Comment:
After submitting a new review, fetch just the reviews data instead of all the data.